Library/ViewModels/PledgeViewModel.swift

```diff
@@ -30,6 +31,13 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
       .map { (project, reward) in
         (reward.minimum, currencySymbol(forCountry: project.country).trimmed())
     }
+
+    self.estimatedDeliveryDate =  projectAndReward.signal
```

Comment:
This signal should probably be tested. Or if we’re merging it with previous signal tests will need to be updated